drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -15,6 +15,7 @@ class SubmitForm < BaseMutation
     def resolve(input:)
       errors = HmisErrors::Errors.new
 
+      ## question: if we can't fnd the form definition or record_class, shouldn't we just raise an exception?
```

Comment:
The pattern in all the mutations right now is to return errors and basically never raise exceptions on purpose. But obviously the frontend needs to be setup to catch unhandled exceptions (and it is) since they will occur anyway.

So yeah, I think it's a very valid point, I don't have a compelling reason as to why we return "validation" errors rather than raising exceptions for problems like this. Just one of those things where we started out with a pattern and ran with it. Happy to change it.